src/interfaces/coral_web/src/cohere-client/client.ts

```diff
@@ -318,8 +322,32 @@ export class CohereClient {
     return body as Deployment[];
   }
 
+  public async listExperimentalFeatures(): Promise<ExperimentalFeatures> {
```

Comment:
toiny nit: since this isn't returning us an array i'd hesitate to use the word `list`
```suggestion
  public async getExperimentalFeatures(): Promise<ExperimentalFeatures> {
```